### ISSUE TITLE
arithmico docs

### ISSUE THREAD

**Issue by cssbue** [behrenle/arithmico#338]
Functionality BBI: The detailed help (docs.arithmico.com) is not yet optimal in terms of accessibility. For example, one jumps from input to output fields, but cannot read the text in between and cannot return to the table of contents, for example.

**Comment by cssbue** [comment]
will be adapted
